<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualKeyboard</class>
 <widget class="QDialog" name="VirtualKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="key_1">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>110</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_9">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>207</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_left">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>307</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_enter">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>15</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>ENTER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>12</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_3">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>13</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_4">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>14</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_6">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>110</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_8">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>111</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_7">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>111</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_backspace">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>111</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>BACKSPACE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_up">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>208</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>↑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_plus">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>209</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_reside">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>209</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_0">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>209</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_down">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>308</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>↓</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_minus">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>309</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_esc">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>309</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>ESC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_right">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>309</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px outset #a0a0a0;
    border-radius: 4px;
}
    QPushButton:pressed
{
	background-color: #D4D4D4;
    color:#610404;
    border: 2px inset #a0a0a0;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
